### DISCUSSION TITLE
IOS H5 uploader 唤起相机录像分辨率很低，这个大家有遇到吗

### DISCUSSION THREAD

**Discussion by tongteng** [youzan/vant#12078] [Q&A]
IOS H5 uploader 唤起相机录像分辨率很低，而且会压缩尺寸，iphone12PM 原相机录制视频宽度是1440*1920，唤起的相机录像的尺寸是360*480

**Comment by inottn** [comment]
你可以试试使用原生唤起相机录像的分辨率

**Reply by tongteng** [reply]
你说的是在APP侧么

**Reply by inottn** [reply]
意思是可以试一下不用组件唤起相机有没有这个问题，需要明确一下这个问题是不是和组件有关系。

**Reply by inottn** [reply]
如果和组件无关的话，你获取解决方法的途径可能会更多

**Reply by tongteng** [reply]
跟组件无关，使用原生input type file 上传也是的，看起来唤起的相机分辨率默认的360p一样，非常模糊，而且无法对焦。

**Reply by tongteng** [reply]
大佬还在么 这个有什么解决方法呢
